lib/host/wasi_nn/wasinnmodule.cpp

```diff
@@ -2,12 +2,20 @@
 // SPDX-FileCopyrightText: 2019-2022 Second State INC
 
 #include "host/wasi_nn/wasinnmodule.h"
+#include "common/errcode.h"
 #include "host/wasi_nn/wasinnfunc.h"
+#include "runtime/hostfunc.h"
+#include "runtime/importobj.h"
+#include "spdlog/sinks/stdout_color_sinks.h"
```

Comment:
Remove this unnecessary include.

Reply:
About adding a special spdlog prefix for `wasi-nn`, should we need to use the `spdlog::stdout_color_mt("wasi-nn")`? If so, this include is necessary.